framework/rbac/AuthorizerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace yii\rbac;
+
+/**
+ * @author Sam Mousa <sam@mousa.nl>
+ * @since 2.0.9
+ */
+interface AuthorizerInterface
```

Comment:
Hmm, `accessChecker`? Not sure I like that better; --> let me know when you think of something ;-)